cmd/server/server.go

```diff
@@ -267,7 +256,7 @@ func checkRequiredConfigs(conf config.Provider) error {
 	return nil
 }
 
-func Initialize(conf config.Provider) error {
+func Initialize(l log.Logger, conf config.Provider) error {
```

Comment:
the logger that we are injecting in this should not use plain formatting similar to cli